renovate.json

```diff
@@ -1,5 +1,30 @@
 {
   "extends": [
     "config:base"
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^dependencies\\.properties$"],
+      "matchStrings": ["=(?<depName>.*?):(?<currentValue>.*?)\\n"],
+      "datasourceTemplate": "maven"
+    },
+    {
+      "fileMatch": ["^dependencies\\.properties$"],
+      "matchStrings": ["version\\.com_google_protobuf=(?<currentValue>.*?)\\n"],
+      "depNameTemplate": "com.google.protobuf:protobuf-java",
+      "datasourceTemplate": "maven"
+    },
+    {
+      "fileMatch": ["^dependencies\\.properties$"],
```

Comment:
I don't think this version is used, it's currently hardcoded [here](https://github.com/googleapis/gax-java/blob/f2b7cb2b6b783095547b798da054b6879d3764f1/build.gradle#L77)

Reply:
It seems to be used by Bazel build though [here](https://github.com/googleapis/gax-java/blob/f2b7cb2b6b783095547b798da054b6879d3764f1/repositories.bzl#L110). We should probably update `gradle.build` to use the property too.

Reply:
I see, thanks! Yes we should, we are removing Gradle soon but this is a quick fix anyway.